### PR TITLE
Finesse new font styles.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review. (I snuck an update to the reacji row's padding in there too.)

### CODE REVIEW

File: shared/chat/conversation/messages/wrapper/index.js [hunk 1]
```diff
@@ -604,7 +606,7 @@ const styles = Styles.styleSheetCreate({
   timestamp: Styles.platformStyles({
     common: {position: 'relative'},
     isElectron: {top: 1},
-    isMobile: {top: 2},
+    isMobile: {top: 1.5},
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/wrapper/index.js]: i doubt we want to do this. why is the half pixel necessary

Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/index.js]: I used an on-screen ruler to line it up. It was half a pixel off either way.

Comment (chrisnojima) [reply shared/chat/conversation/messages/wrapper/index.js]: i think this is indicative of a parent container thats not the right size and then doing a centering

Comment (adamjspooner) [reply shared/chat/conversation/messages/wrapper/index.js]: Okay, I'll see what I can do to make this better. 👍

**Comment by adamjspooner** [comment]
@chrisnojima, how's that?

**Comment by adamjspooner** [comment]
Totally! Thanks! 🙌

**Pull request merged**